extension/src/externalresources/ExternalResources.ts

```diff
@@ -139,4 +163,8 @@ export class BlobContainer implements BlobContainerInterface {
   public url(name: string): URL {
     return new URL(`${this.baseUrl}${name}?${this.sasToken}`);
   }
+
+  public tokenIsValid(): boolean {
```

Comment:
is this used?

Reply:
Not anymore. I was planning to use it to throw an error  when initiating `BlobContainer`. But that would actually cause the extension activation to break. I think you track it by commit messages if your curious. 

Removed function.